ros_buildfarm/bloom_status.py

```diff
@@ -121,7 +140,7 @@ def _query_package_statuses(g, package_info):
             d_info['status'] = status
 
 
-def build_bloom_status_page(distros=None, output_dir='.'):
+def build_bloom_status_page(packages=[], distros=None, output_dir='.'):
```

Comment:
As a best practice, never put mutable objects as the default of a keyword argument. Python does something fun where it inits the instance once and re-uses it across every invocation of that function. Typically I just use a `tuple()` instead because it matches `list`'s duck type.

This isn't a big deal here but it would burn you bad if you later did something to `packages` and wondered why those changes accumulated across invocations of the function.